### PR TITLE
systemvm: document workaround for older ACS env with systemvms

### PR DESCRIPTION
**Pull request by rhtyd**

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan docbuild

File: source/upgrading/upgrade/_sysvm_restart.rst [hunk 1]
```diff
@@ -26,6 +26,22 @@ from the new systemVM template version.
    or do it by using third-party tools such as Ansible.
    Below we are giving instructions for using the "cloudstack-sysvmadm" script.
 
+Note: When upgrading from older CloudStack environments (for example, 4.9-4.13)
+with XenServer or XCP-ng hosts, some systemvms and virtual routers may not have
+ejected or removed the older systemvm.iso. The old systemvm.iso needs to be
+ejected and removed before new systemvms and virtual routers are (re-)created).
+This issue has been fixed in CloudStack 4.15, upgrading from older enviroments
```
Comment (shwstppr) [inline source/upgrading/upgrade/_sysvm_restart.rst:33]:
```suggestion
This issue has been fixed in CloudStack 4.15, upgrading from older environments
```

**Comment by rhtyd** [comment]
@blueorangutan docbuild

**Review by borisstoyanov — APPROVED** [review]
LGTM, tested the given workaround, happens with 4.9, 4.11, 4.13

**Comment by rhtyd** [comment]
This is not needed, let's fix in code https://github.com/apache/cloudstack/pull/5053/files

**Pull request closed without merging**